src/pages/MyReports/components/Content/Content.tsx

```diff
@@ -22,7 +22,7 @@ const Content: FC = () => (
         </div>
       </Button>
       <Button className="w-[250px] h-[90px] hover:bg-hover">Browse Templates</Button>
-      <Link to="/my-reports/history">
+      <Link to="/user/my-reports/history">
```

Comment:
I guess this will not working in GitHub pages

Reply:
preview looks ok